modules/javafx.graphics/src/main/java/javafx/scene/input/Dragboard.java

```diff
@@ -38,20 +37,12 @@
  */
 public final class Dragboard extends Clipboard {
 
-    /**
-     * Whether access to the data requires a permission.
-     */
-    private boolean dataAccessRestricted = true;
-
     Dragboard(TKClipboard peer) {
         super(peer);
     }
 
     @Override
     Object getContentImpl(DataFormat dataFormat) {
-        if (dataAccessRestricted) {
```

Comment:
this method can be deleted here

Reply:
Oh, I see what you mean now. You mean that `Dragboard` no longer needs to override this method, since all it does now is call `super`. I'll remove it.